src/app/public/modules/text-editor/text-editor.component.spec.ts

```diff
@@ -300,17 +306,19 @@ describe('Text editor', () => {
   }
   //#endregion
 
-  beforeEach(() => {
+  beforeEach(async(() => {
```

Comment:
not sure why this diff is so long. all ive done here is:
- restructure the unit tests to be in two categories:
     1. standard
     2. reactive
- add the 2 new tests for disabling